README.md

```diff
@@ -31,6 +31,7 @@ These guidelines are designed exclusively to help the Working Group achieve its
 The Security Working Group has a few communication channels:
```

Comment:
```suggestion
* Catch the announcements on ROS Discourse using the [`wg-security` tag](https://discourse.ros.org/tags/wg-security)
* Chat with us in the [Matrix room](https://matrix.to/#/!LcRLnAIRWjSCfZmMeD:matrix.org?via=matrix.org)
* Tag us on Github using @ros-security-wg
* Email us on the [ROS Security Working Group mailing list](https://groups.google.com/forum/#!forum/ros-security), which is used as a point of contact to reach the WG (e.g. when we get mentioned on GitHub
```

Reply:
+1

Reply:
8b4d11b should capture these--I also shifted this list to third person to be consistent with the rest of the document.